src/Matmar10/Bundle/MoneyBundle/DBAL/Type/CurrencyType.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Matmar10\Bundle\MoneyBundle\DBAL\Type;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\Type;
+use Matmar10\Bundle\MoneyBundle\Service\CurrencyManager;
+use Matmar10\Money\Entity\CurrencyInterface;
+use Matmar10\Money\Exception\InvalidArgumentException;
+
+/**
+ * Description of CurrencyType
+ *
+ * @author Volker von Hoesslin <volker.von.hoesslin@empora.com>
+ */
+class CurrencyType extends Type {
```

Comment:
thx, the only down side is that doctrine types dosnt support services... because of that this hack here: https://github.com/vvh-empora/money-bundle/commit/87a13d3f220c158850e5c8eb13797c8c94cbab10#diff-282d7d1db419cd91dbb5f9b0e2123fa3R31